src/main/scala/it/pagopa/interop/authorizationprocess/api/impl/ClientApiServiceImpl.scala

```diff
@@ -54,23 +55,18 @@ final case class ClientApiServiceImpl(
 
   def internalServerError(responseProblem: Problem)(implicit
     toEntityMarshallerProblem: ToEntityMarshaller[Problem]
-  ): Route =
-    complete((500, responseProblem))
+  ): Route = complete((500, responseProblem))
 
-  /** Code: 201, Message: Client created, DataType: Client
-    * Code: 401, Message: Unauthorized, DataType: Problem
-    * Code: 404, Message: Not Found, DataType: Problem
-    * Code: 500, Message: Internal Server Error, DataType: Problem
-    */
   override def createConsumerClient(clientSeed: ClientSeed)(implicit
     contexts: Seq[(String, String)],
     toEntityMarshallerProblem: ToEntityMarshaller[Problem],
     toEntityMarshallerClient: ToEntityMarshaller[Client]
   ): Route = authorize(ADMIN_ROLE) {
-    logger.info("Creating CONSUMER client {} for and consumer {}", clientSeed.name, clientSeed.consumerId)
     val result = for {
+      organizationId <- getClaimFuture(contexts, ORGANIZATION_ID_CLAIM).flatMap(_.toFutureUUID)
+      _ = logger.info("Creating CONSUMER client {} for and consumer {}", clientSeed.name, organizationId)
```

Comment:
I get why you moved the log here, but probably I would prefer to have a partial log at the beginning.
So just something like this would be fine
```
logger.info("Creating CONSUMER client {}", clientSeed.name)
```

The uglier but useful solution should have both